src/js/editing/Typing.js

```diff
@@ -45,14 +46,23 @@ define([
       var nextPara;
       // on paragraph: split paragraph
       if (splitRoot) {
-        nextPara = dom.splitTree(splitRoot, rng.getStartPoint());
+        // if new line has content (not a line break)
+        if (!/^BR/i.test(splitRoot.firstChild.nodeName)) {
+          nextPara = dom.splitTree(splitRoot, rng.getStartPoint());
 
-        var emptyAnchors = dom.listDescendant(splitRoot, dom.isEmptyAnchor);
-        emptyAnchors = emptyAnchors.concat(dom.listDescendant(nextPara, dom.isEmptyAnchor));
+          var emptyAnchors = dom.listDescendant(splitRoot, dom.isEmptyAnchor);
+          emptyAnchors = emptyAnchors.concat(dom.listDescendant(nextPara, dom.isEmptyAnchor));
 
-        $.each(emptyAnchors, function (idx, anchor) {
-          dom.remove(anchor);
-        });
+          $.each(emptyAnchors, function (idx, anchor) {
+            dom.remove(anchor);
+          });
+        // if it is an empty line
+        } else {
+          // disable UL/OL and escape!
+          var bullet = new Bullet();
```

Comment:
Hm, I really thought this variable wasn't useful enough to make it a member in the module. Btw, it would be awesome if I could toggle off the list without using this stuff (I still think it doesn't belong to the Typing module scope).